src/widgets/upload/upload.js

```diff
@@ -390,6 +399,26 @@ class Upload extends React.Component<UploadProps, StateProps> {
     return fileListDone;
   };
 
+  customUploadStatus = (type: string, hashMark: string) => {
+    const { fileListDone } = this.state;
+    switch (type) {
```

Comment:
function  doUpload(x1, x2){

const list = this.updateFieldList(fileListDone, hashMark, [
        x1,
        ]);
        this.setStateValue(x2);

}

case 'success': 

  doUploader(x1, x2);

case xxx
doUploader(x1,x2)